### PR TITLE
feat: allow the renderer to have a label object property

### PR DESCRIPTION
**Pull request by lpatiny**

### CODE REVIEW

**Comment by hamed-musallam** [comment]
We should consider adding an additional property for toggling the visibility of the label. While it is currently possible to control this by not passing the label but i think having a separate property dedicated to this functionality would be better.

**Comment by targos** [comment]
I disagree, unless you're suggesting that the passed label still has some effect on the result and we just be invisible.

**Comment by hamed-musallam** [comment]
> I disagree, unless you're suggesting that the passed label still has some effect on the result and we just be invisible.

We use this component in various locations, it's not necessary for all our use cases to have a label underneath. on the other hand, to keep the consistency, we have other properties, such as the option to show/hide the atom number ...etc

File: types.d.ts [hunk 1]
```diff
@@ -28,6 +28,15 @@ export interface IBaseSvgRendererProps extends IMoleculeToSVGOptions {
...
   onBondClick?: (bondId: number, event: MouseEvent<SVGElement>) => void;
+
+  autoCrop?: boolean;
+  autoCropMargin?: number;
+
+  label?: {
+    fontFamily?: string;
+    fontSize?: string;
+    value?: string;
+  };
```
Comment (targos) [inline types.d.ts:39]: sub-objects are not usual in React props. And here I find it inconsistent with the other options like bond highlight that are separated in multiple props.

File: src/components/SvgRenderer.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default function SvgRenderer(props) {
   };
   const serializedOptions = JSON.stringify(toSVGOptions);
   const svgString = useMemo(() => {
-    return mol.toSVG(width, height, id, JSON.parse(serializedOptions));
+    return getSVG(mol, width, height, id, serializedOptions)
```
Comment (targos) [inline src/components/SvgRenderer.js:50]: You're mixing the options passed to `mol.toSVG` and the label options

**Pull request merged**